app/src/androidTest/java/org/akvo/flow/ApplicationTest.java

```diff
@@ -1,3 +1,22 @@
+/*
+* Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
```

Comment:
Same comment regarding adding copyright for only the years when the file was actually edited. this should be starting in 2017 not 2010 (or rather look for the year file was first created). Check the rest of the files.